### DISCUSSION TITLE
Application appearance

### DISCUSSION THREAD

**Discussion by dh3b** [martinet101/ElevenClock#568] [Q&A] [answered]
Hey!
Does anyone know how can I make a translucent/semitransparent background in python application that looks like eleven clock? I really like how the app is made and I wanted to make my app look like this too.

I would be appreciated if someone would answer.

**Comment by marticliment** [comment, marked as answer]
Take a look at this: https://github.com/martinet101/ElevenClock/wiki
If something isn't clear, just tell it and i'll add more details.

**Reply by dh3b** [reply]
I get that you were using PyQt5 for making the gui. I saw the part of the code 
`from external.blurwindow import ExtendFrameIntoClientArea, GlobalBlur`
that imports global blur, does this library makes the background blurry and if so how do i use it?

**Reply by marticliment** [reply]
Yes, it does. It is actually a modified verion of this: https://github.com/Peticali/PythonBlurBehind.

**Reply by dh3b** [reply]
Thank you very much and have a good day!
